### PR TITLE
Web

### PR DESCRIPTION
**Pull request by Mahmoud-nfz**

### CODE REVIEW

File: web/src/app/_components/create-post.tsx
Comment (omar-besbes) [inline web/src/app/_components/create-post.tsx:1]: This component is not used. Could you remove it ?
Remove `_components` dir altogether and let's keep one single components dir.

File: web/src/app/players/[id]/compare/[id2]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                layout="fill" // Changed to 'fill' to better control positioning
+                objectFit="contain" // Keeps the image within the bounds of the div without stretching
+                objectPosition="bottom" // Aligns the image to the bottom of the div
+              />
+            </div>
+            <div
+              style={{
+                width: "300px",
+                height: "600px",
+                display: "flex",
```
Comment (omar-besbes) [inline web/src/app/players/[id]/compare/[id2]/page.tsx:44]: Why do you love inline styles so much ?? It's a bad practice and they are hard to override.
And they clutter the code.

File: web/Dockerfile
Comment (omar-besbes) [inline web/Dockerfile:1]: This Dockerfile is not working

File: docker-compose.yaml [hunk 2]
```diff
@@ -1,4 +1,14 @@
 services:
+  web:
+    platform: "linux/amd64"
+    build:
+      context: ./web
+      dockerfile: Dockerfile
+    working_dir: /app
```
Comment (omar-besbes) [inline docker-compose.yaml]: No need to specify `dockerfile` and `working_dir`. `dockerfile` is by default set to `Dockerfile` and `working_dir` is already set to `/app` in the Dockerfile.
Also, consider adding `environment`.

File: web/src/server/db/db.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import * as r from "rethinkdb";
+
+// Object to store the connection
+const dbConfig = {
+  host: "localhost",
+  port: 28015,
```
Comment (omar-besbes) [inline web/src/server/db/db.ts]: You already have, an `env.js` to handle your conifgs. Please, move these inside `env.js`!

**Review by omar-besbes — CHANGES_REQUESTED** [review]
That about sums it up. Good job!

File: web/Dockerfile
Comment (omar-besbes) [reply web/Dockerfile:1]: Here is the error that comes up: https://pastebin.com/8UyJWWm8

File: web/src/app/players/[id]/compare/[id2]/page.tsx (hunk 1 above)
Comment (Mahmoud-nfz) [reply web/src/app/players/[id]/compare/[id2]/page.tsx:44]: It is what it is :(

**Review by omar-besbes — APPROVED** [review]

**Pull request merged**